docs: how to make Keptn work with vCluster (#2382)
keptn/lifecycle-toolkit: docs/content/en/docs/install/reqs.md

Server Version: v1.24.0
```

> Keptn scheduling is implemented using the Kubernetes Scheduler Framework
> when running on Kubernetes versions 1.26 and earlier.
> For Kubernetes versions greater than 1.26,
> Keptn scheduling is implemented using Kubernetes Feature gates,
> unless the `schedulingGatesEnabled` Keptn Helm chart value is set to `false`.
> See
> [Keptn integration with Scheduling](../architecture/components/scheduler/)
> for details.
> 
> Note that you must modify your vCluster configuration
> to work with the Kubernetes Scheduler Framework.
> See
> [Running Keptn with vCluster](install.md/#running-keptn-with-vcluster)
> for more information.

Reviewer: What do you think of this?
```suggestion
Keptn makes use of a custom scheduler
when running on Kubernetes v1.26 and earlier.
For Kubernetes v1.27 and later, scheduling is
implemented using [Kubernetes scheduling gates](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness/),
unless the `schedulingGatesEnabled` Helm value is set to `false`.
See
[Keptn integration with Scheduling](../architecture/components/scheduler/)
for details.

If Keptn is installed on a [vCluster](https://www.vcluster.com/) with
Kubernetes v1.26 or earlier, some extra configuration
needs to be added for full compatibility.
See
[Running Keptn with vCluster](install.md/#running-keptn-with-vcluster)
for more information.
```

Server Version: v1.24.0
```

Keptn uses a different scheduling method
when using Kubernetes versions 1.26 and older

> or if the `schedulingGatesEnabled` Helm chart value is set to `false`

Reviewer: ```suggestion
or if the `schedulingGatesEnabled` Helm value is set to `false`
```

> implemented using [Kubernetes scheduling gates](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness/),

Author: ```suggestion
implemented using
[Kubernetes scheduling gates](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness/),
```

 See
[Keptn integration with Scheduling](../architecture/components/scheduler/)
for details.

> Note that you must modify your vCluster configuration

Reviewer: This paragraph almost seems like a duplicate of the previous one 🤔 
Maybe they can be combined?

Author: They are two different statements, actually, but they do look similar.  It's because of the long string about the Helm chart value that implements the "old" scheduler...  And I don't know how to say that more succinctly.  So the logical flow for the section is:

* Keptn works on all k8s 0.24 and later
* The scheduler implementation is different on 0.26 and later, and this links to the Architecture section about scheduling
* vCluster must be modified for k8s before 0.26, and links to instructions for doing this.

Is there a succinct name for the two schedulers?

Author: Okay, I tried something different.  I may have used the wrong terminology but see if this works better, and if you can correct my terminology ;-)
